packages/ddp-client/livedata_connection_tests.js

```diff
@@ -95,6 +95,92 @@ Tinytest.add("livedata stub - receive data", function (test) {
   test.isUndefined(conn._updatesForUnknownStores[coll_name]);
 });
 
+Tinytest.add("livedata stub - buffering data", function (test) {
```

Comment:
I could wrap this in a `Meteor.isServer` to avoid the error happening on client tests, however I thought this might eventually (soon?) be ported over to using `emcascript`, which might allow this test to work there too.  Let me know if you want me to add it now.